docs/content/setup/docker.md

```diff
@@ -14,12 +14,22 @@ The Infection Monkey Docker container works on Linux only. It is not compatible
 ## Deployment
 
 ### 1. Load the docker images
+#### Mongo Database Docker image
 1. Pull the MongoDB v6.0 Docker image:
 
     ```bash
     sudo docker pull mongo:6.0
     ```
 
+#### Monkey Island Docker image
+1. Pull the Monkey Island Docker image through DockerHub:
+
+    ```bash
+    sudo docker pull infectionmonkey/monkey_island:latest
+    ```
+
+or
+
 1. Extract the Monkey Island Docker tarball:
```

Comment:
So we're keeping the tarball option?

Reply:
I am not sure about it, so I am keeping it as an option.

Reply:
Let's remove it. It complicates the rest of the steps. Can we just move it to a developer documentation (README or something) somewhere so that we still have a reference?